src/community/develop-a-component-or-pattern/index.md.njk

```diff
@@ -30,27 +29,37 @@ If you’re happy to go ahead, the team will assign you to the issue as a contri
 
 ## Research and develop your contribution
 
-While you’re working on your contribution, the Design System team will arrange a weekly catch up to find out how you’re getting on and if you need any help. 
+While you’re working on your contribution, the Design System team will arrange a weekly catch up to find out how you’re getting on and if you need any support.
+
+The following principles will help you research and develop your contribution.
+
+### Start with what exists
 
 Find examples of the component or pattern already in use. The best way to do this is to ask the government design community on the [digital service designers mailing list](https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en-GB#!forum/digital-service-designers) or the [#design channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=design).
 
 Examples and research from government services are usually most relevant. But look at how other organisations solve the problem too.
 
-Involve people from the [government design community](https://www.gov.uk/service-manual/communities/design-community) in any work you’re doing. This makes it less likely that you’ll spend time doing work or research that’s already been done. 
+### Work with the community
+
+Involve people from the [government design community](https://www.gov.uk/service-manual/communities/design-community) in any work you’re doing. This makes it less likely that you’ll spend time doing work or research that’s already been done.
 
 By finding people who are working on similar solutions, you can benefit from progress they’ve made and relevant research they’ve done.
 
 Once you have collected some examples and research findings, you should start designing a specific implementation of the component or pattern to research with.
 
-Ideally you’ll research as part of an actual service, but it’s possible to test using prototypes as long as they are realistic. Work with your own team or find a team to work with through the design community. 
+Ideally you’ll research as part of an actual service, but it’s possible to test using prototypes as long as they are realistic. Work with your own team or find a team to work with through the design community.
+
+### Test with users
```

Comment:
@calvin-lau-sig7 Wonder should re retitle this 'Test for accessibility' (or similar), as the sentence that follows mentions criteria, not users.

Reply:
```suggestion
### Test for accessibility
```